app/policies/user_policy.rb

```diff
@@ -0,0 +1,38 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
+    user_authorized?
+  end
+
+  def feed?
+    true
+  end
+
+  def discover?
+    true
```

Comment:
probably should be `user == current_user`

Reply:
fixed! thank you!